features/utils.py

```diff
@@ -113,15 +121,16 @@ def go_to_home_page(browser):
     home_page.flaw_list_exist()
 
 
-def go_to_specific_flaw_detail_page(browser, cve_id=TEST_FLAW_CVE_ID):
+def go_to_specific_flaw_detail_page(browser, embargoed=False):
```

Comment:
The other file calls this function, e.g., Line 118 of flaw_filter.py, that should be updated together.